src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -266,17 +275,21 @@ public void Load(string settingsXml)
 #endif
 
 #if DEBUG && VERBOSE
-            // Force Verbosity to 1 under Debug
+// Force Verbosity to 1 under Debug
             Verbosity = 1;
 #endif
 
-            var inProcDataCollectorNode = doc.SelectSingleNode("RunSettings/InProcDataCollectionRunSettings/InProcDataCollectors");
-            InProcDataCollectorsAvailable = inProcDataCollectorNode != null && inProcDataCollectorNode.SelectNodes("InProcDataCollector").Count > 0;
+            var inProcDataCollectorNode =
+                doc.SelectSingleNode("RunSettings/InProcDataCollectionRunSettings/InProcDataCollectors");
+            InProcDataCollectorsAvailable = inProcDataCollectorNode != null &&
+                                            inProcDataCollectorNode.SelectNodes("InProcDataCollector").Count > 0;
```

Comment:
(Seems like a lot of changes are just rewrapping? I usually do formatting in a separate commit from the logical changes.)